go/vt/vttablet/tabletmanager/vreplication/vreplicator.go

```diff
@@ -500,6 +505,32 @@ func (vr *vreplicator) throttlerAppName() string {
 	return strings.Join(names, ":")
 }
 
+func (vr *vreplicator) updateTimeThrottled(componentThrottled string) error {
```

Comment:
I feel we it may be worth having constants defined for the components, e.g. `vreplication.Compenent` type (alias for string) with a value of e.g. `vreplication.VPlayerComponent` being a const string for "vplayer"? Strings introduces the opportunity for typos, invalid components, etc.

Reply:
Fixed. Now formalized as `VReplicationComponent`.

Reply:
Rather, `ComponentName`.